modules/swagger-codegen/src/main/java/com/wordnik/swagger/codegen/examples/ExampleGenerator.java

```diff
@@ -53,12 +65,10 @@ else if(property != null && mediaType.startsWith("application/xml")) {
       }
     }
     else {
-      for(String key: examples.keySet()) {
-        String value = examples.get(key);
-
-        Map<String, String> kv = new HashMap<String, String>();
-        kv.put("contentType", key);
-        kv.put("example", value);
+      for(Map.Entry<String, Object> entry: examples.entrySet()) {
+        final Map<String, String> kv = new HashMap<String, String>();
+        kv.put("contentType", entry.getKey());
+        kv.put("example", Json.pretty(entry.getValue()));
```

Comment:
@wing328, I would like to rework this piece as a part of this PR. I realized the problem with `contentType` after the PR had been created. Perhaps @fehguy or @webron could provide some additional comments here.